k3d/factory.py

```diff
@@ -1587,6 +1587,8 @@ def volume(
         focal_length=0.0,
         focal_plane=100.0,
         ray_samples_count=16,
+        mask=[],
```

Comment:
mutable default value

Reply:
We have that in so many places ;)

Reply:
I know, I know...
Easiest fix (if it works!) -- use empty tuple.
Maybe it's benign.
But maybe, when it becomes a traitlet of a Drawable, and gets modified later, then the modified version becomes the new default value ;)
(I don't know if Traitlet does a copy, or keeps a reference)